fgpyo/sam/builder.py

```diff
@@ -116,14 +119,27 @@ def __init__(
             extra_header: a dictionary of extra values to add to the header, None otherwise.  See
                           `::class::~pysam.AlignmentHeader` for more details.
             seed: a seed value for random number/string generation
+            sort_order: optional sort order, if `None` reads will be output in the same order as
+                they were appended. If `SamOrder.Coordinate`, reads will be ordered by reference
+                index and coordinate order. If `SamOrder.QueryName`, reads will be ordered by
+                query name. `SamOrder.TemplateCoordinate` is not currently supported.
```

Comment:
1. lets remove `TemplateCoordinate` it until it is supported
2. I don't think you need to explain the options here, you have the type, and folks can look that up